src/Nethermind/Nethermind.Evm/AccessTracker.cs

```diff
@@ -0,0 +1,89 @@
+// SPDX-FileCopyrightText: 2022 Demerzel Solutions Limited
+// SPDX-License-Identifier: LGPL-3.0-only
+
+using System;
+using System.Collections.Generic;
+using Nethermind.Core;
+using Nethermind.Core.Collections;
+using Nethermind.Core.Eip2930;
+using Nethermind.Int256;
+
+namespace Nethermind.Evm
+{
+    public sealed class AccessTracker
+    {
+        public JournalSet<Address> AccessedAddresses { get; }
+        public JournalSet<StorageCell> AccessedStorageCells { get; }
+        public JournalCollection<LogEntry> Logs { get; }
+        public JournalSet<Address> DestroyList { get; }
+        public HashSet<AddressAsKey> CreateList { get; }
```

Comment:
If they are public should they be `IReadOnlySet`?

Reply:
All made into `IReadOnlySet`